### PR TITLE
Add functionality to buttons

### LINKED ISSUE #13: Hide home screen buttons when report button is clicked

### PR DESCRIPTION
**Pull request by AshleyLTW**
Closes #13 

The buttons to show/hide the menu and report screen now work. I also changed how we hide report/menu on load to prevent users from seeing them flash before the javascript is called.

There are some double quotation marks in the script.js file but I didn't change them since I think they're changed in another PR.

### CODE REVIEW

File: web-app/src/main/webapp/script.js [hunk 1]
```diff
@@ -57,10 +57,33 @@ function showMessageOnInfoWindow(message, position, map, infoWindow) {
...
   document.getElementById('report-button').addEventListener('click', showReportForm);
+  document.getElementById('back-icon').addEventListener('click', hideReportForm);
+  document.getElementById('menu-button').addEventListener('click', showMenu);
+  document.getElementById('close-menu').addEventListener('click', hideMenu);
 };
 
 function showReportForm() {
-  document.getElementById("form-container").style.display = "block";
+  document.getElementById('form-container').style.display = 'block';
+  let homeElements = document.getElementsByClassName('home');
```
Comment (jiminbyun) [inline web-app/src/main/webapp/script.js]: nit: const?

File: web-app/src/main/webapp/script.js [hunk 2]
```diff
@@ -57,10 +57,33 @@ function showMessageOnInfoWindow(message, position, map, infoWindow) {
...
+  for (let i = 0; i < homeElements.length; i++) {
```
Comment (jiminbyun) [inline web-app/src/main/webapp/script.js]: nit: for (element in homeElements) ?

File: web-app/src/main/webapp/script.js [hunk 3]
```diff
@@ -57,10 +57,33 @@ function showMessageOnInfoWindow(message, position, map, infoWindow) {
...
+    homeElements[i].style.display = 'block';
+  }
+}
+
+function showMenu() {
+  document.getElementById('menu').style.display = 'block';
+}
+
+function hideMenu() {
+  console.log('hiding')
```
Comment (jiminbyun) [inline web-app/src/main/webapp/script.js]: nit: Remove debugging message

File: web-app/src/main/webapp/script.js [hunk 4]
```diff
@@ -57,10 +57,33 @@ function showMessageOnInfoWindow(message, position, map, infoWindow) {
...
+  document.getElementById('menu').style.display = 'none';
```
Comment (jiminbyun) [inline web-app/src/main/webapp/script.js]: nit: For these one liner functions, can you just make it as a arrow function in onload instead??

**Review by jiminbyun — APPROVED** [review]

File: web-app/src/main/webapp/script.js (hunk 2 above)
Comment (AshleyLTW) [reply web-app/src/main/webapp/script.js]: I don't think this works for nodeLists, I keep getting all the indices first before the element

Comment (darrnshn) [reply web-app/src/main/webapp/script.js]: Should be possible: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of (see "Iterating over a DOM collection")

File: web-app/src/main/webapp/script.js [hunk 5]
```diff
@@ -57,10 +57,26 @@ function showMessageOnInfoWindow(message, position, map, infoWindow) {
...
+  const homeElements = document.getElementsByClassName('home');
+  for (let i = 0; i < homeElements.length; i++) {
+    homeElements[i].style.display = 'none';
+  }
+}
+
+function hideReportForm() {
+  document.getElementById('form-container').style.display = 'none';
+  let homeElements = document.getElementsByClassName('home');
+  for (let i = 0; i < homeElements.length; i++) {
```
Comment (darrnshn) [inline web-app/src/main/webapp/script.js]: nit: same here

File: web-app/src/main/webapp/script.js [hunk 6]
```diff
@@ -57,10 +57,26 @@ function showMessageOnInfoWindow(message, position, map, infoWindow) {
...
+  document.getElementById('form-container').style.display = 'block';
+  const homeElements = document.getElementsByClassName('home');
+  for (let i = 0; i < homeElements.length; i++) {
+    homeElements[i].style.display = 'none';
+  }
+}
+
+function hideReportForm() {
+  document.getElementById('form-container').style.display = 'none';
+  let homeElements = document.getElementsByClassName('home');
```
Comment (darrnshn) [inline web-app/src/main/webapp/script.js]: nit: const

**Review by darrnshn — APPROVED** [review]

**Pull request merged**